### PR TITLE
Add Javadoc for Money.

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**

### CODE REVIEW

File: src/main/java/com/teamdev/javaclasses/aleksandrov/money/Money.java [hunk 1]
```diff
@@ -99,7 +99,7 @@ public static Money newAmount(String currencyCode, long amount) {
     }
 
     /**
-     * Compares amounts of moneys in case if they have the same currency.
+     * Compares  amounts of moneys in case if they have the same currency.
      *
      * @param o {@link Money} object
      * @return itn value less than zero in case if comparable is bigger, zero if equals, and more than zero if bigger
```
Comment (alexander-yevsyukov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/money/Money.java]: You don't need to describe a type of a returned value. We can see it from the method signature.

”Less than zero” is called ”negative”. ”More than zero” is called positive. Look at any `Comparable` in Guava or another well-known library. How do they describe such things? What words do they use?

File: src/main/java/com/teamdev/javaclasses/aleksandrov/money/Money.java [hunk 2]
```diff
@@ -20,7 +20,7 @@
 package com.teamdev.javaclasses.aleksandrov.money;
 
 /**
- * Class that allows to work with money as value object.
+ * Class  that allows to work with money as value object.
```
Comment (alexander-aleksandrov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/money/Money.java]: Fixed

File: src/main/java/com/teamdev/javaclasses/aleksandrov/money/Money.java (hunk 1 above)
Comment (alexander-aleksandrov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/money/Money.java]: Fixed

File: src/main/java/com/teamdev/javaclasses/aleksandrov/money/Money.java [hunk 3]
```diff
@@ -43,7 +43,7 @@ public Currency getCurrency() {
     }
 
     /**
-     * Gets amount of money.
+     * Gets  amount of money.
```
Comment (alexander-aleksandrov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/money/Money.java]: Fixed

File: src/main/java/com/teamdev/javaclasses/aleksandrov/money/Currency.java [hunk 4]
```diff
@@ -20,17 +20,17 @@
...
 /**
- * Represents different currencies.
+ * Represents different  currencies.
  *
  * @author Alexander Aleksandrov
  */
 public enum Currency {
     /**
-     * Dollars of United States of America
+     * Dollars  of United States of America
```
Comment (alexander-aleksandrov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/money/Currency.java]: Fixed

File: src/main/java/com/teamdev/javaclasses/aleksandrov/money/Currency.java [hunk 5]
```diff
@@ -20,17 +20,17 @@
 package com.teamdev.javaclasses.aleksandrov.money;
 
 /**
- * Represents different currencies.
+ * Represents different  currencies.
```
Comment (alexander-aleksandrov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/money/Currency.java]: Fixed

File: src/main/java/com/teamdev/javaclasses/aleksandrov/money/Money.java [hunk 6]
```diff
@@ -34,7 +34,7 @@
     }
```
Comment (alexander-aleksandrov) [inline src/main/java/com/teamdev/javaclasses/aleksandrov/money/Money.java]: Fixed

**Pull request merged**